examples/single_column_radiation.jl

```diff
@@ -64,60 +86,97 @@ q₀ = 0.015    # surface specific humidity (kg/kg)
 
 # ## Visualization
 #
 # After `set!`, the radiation has been computed. We build Fields and
 # AbstractOperations to visualize the atmospheric state and radiative fluxes.
 
-T = model.temperature
+T = gray_model.temperature
 pᵣ = reference_state.pressure
-qᵗ = model.specific_moisture
-qˡ = model.microphysical_fields.qˡ
-ℋ = RelativeHumidityField(model)
-
-ℐ_lw_up = radiation.upwelling_longwave_flux
-ℐ_lw_dn = radiation.downwelling_longwave_flux
-ℐ_sw = radiation.downwelling_shortwave_flux
-ℐ_net = ℐ_lw_up + ℐ_lw_dn + ℐ_sw
-
-set_theme!(fontsize=14, linewidth=3)
-fig = Figure(size=(1200, 400), fontsize=14)
-
-ax_T = Axis(fig[2, 1]; xlabel="Temperature, T (K)", ylabel="Altitude (km)")
-ax_p = Axis(fig[2, 2]; xlabel="Pressure, p (hPa)")
-ax_q = Axis(fig[2, 3]; xlabel="Specific humidity, q (kg/kg)")
-ax_H = Axis(fig[2, 4]; xlabel="Relative humidity, ℋ (%)")
-ax_I = Axis(fig[2, 5:6], xlabel="Radiation intensity, ℐ (W/m²)",
-            ylabel="Altitude (km)", yaxisposition=:right)
-
-[hideydecorations!(ax, grid=false) for ax in (ax_p, ax_q, ax_H)]
-hidespines!(ax_T, :r, :t)
-hidespines!(ax_p, :l, :r, :t)
-hidespines!(ax_q, :l, :r, :t)
-hidespines!(ax_H, :l, :r, :t)
-hidespines!(ax_I, :l, :t)
-
-
-lines!(ax_T, T)
-lines!(ax_p, pᵣ / 100)  # Convert Pa to hPa
-
-lines!(ax_q, qᵗ; label="qᵗ (total)")
-lines!(ax_q, qˡ; label="qˡ (liquid)")
-axislegend(ax_q, position=:rt, framevisible=false)
-
-lines!(ax_H, 100ℋ)  # Convert to %
-
-# All radiation fluxes in one panel (positive = upward, negative = downward)
-lines!(ax_I, ℐ_lw_up; label="LW ↑")
-lines!(ax_I, ℐ_lw_dn; label="LW ↓")
-lines!(ax_I, ℐ_sw; linestyle=:dash, label="SW ↓")
-lines!(ax_I, ℐ_net; linewidth=4, alpha=0.5, color=:black, label="Net")
-
-Legend(fig[1, 6], ax_I, orientation=:horizontal, nbanks=2, framevisible=false)
-
-title = "Single Column Gray Radiation with O'Gorman & Schneider (2008) optical thickness"
-fig[1, :] = Label(fig, title, fontsize=18, tellwidth=false)
+qᵗ = gray_model.specific_moisture
+ℋ = RelativeHumidityField(gray_model)
+
+ℐ_lw_up_gray = gray_radiation.upwelling_longwave_flux
+ℐ_lw_dn_gray = gray_radiation.downwelling_longwave_flux
+ℐ_sw_gray = gray_radiation.downwelling_shortwave_flux
+ℐ_net_gray = ℐ_lw_up_gray + ℐ_lw_dn_gray + ℐ_sw_gray
+
+ℐ_lw_up_clear = clear_sky_radiation.upwelling_longwave_flux
+ℐ_lw_dn_clear = clear_sky_radiation.downwelling_longwave_flux
+ℐ_sw_clear = clear_sky_radiation.downwelling_shortwave_flux
+ℐ_net_clear = ℐ_lw_up_clear + ℐ_lw_dn_clear + ℐ_sw_clear
+
+ℐ_lw_up_2xco2 = high_co2_radiation.upwelling_longwave_flux
+ℐ_lw_dn_2xco2 = high_co2_radiation.downwelling_longwave_flux
+ℐ_sw_2xco2 = high_co2_radiation.downwelling_shortwave_flux
+ℐ_net_2xco2 = ℐ_lw_up_2xco2 + ℐ_lw_dn_2xco2 + ℐ_sw_2xco2
+
+set_theme!(fontsize=14, linewidth=2.5)
+
+# Format altitude ticks in km (but keep internal units in meters).
+z_ticks_km = 0:5:20
+z_ticks_m = ((z_ticks_km .* 1000), string.(z_ticks_km))
+
+fig = Figure(size=(1400, 800), fontsize=14)
+
```

Comment:
The `fig` above is producing a blank plot at this point: https://numericalearth.github.io/BreezeDocumentation/previews/PR334/literated/single_column_radiation/#Visualization
```suggestion
nothing #hide

```